cosmoz-data-nav.js

```diff
@@ -3,7 +3,17 @@
 
 (function () {
 	'use strict';
-	const IS_V2 = Polymer.flush != null;
+	const _async = window.requestIdleCallback || window.requestAnimationFrame || Polymer.Base.async,
+		_asyncPeriod = function (cb, minimum = 5) {
+			return _async(function (deadline) {
+				if (deadline && 'IdleDeadline' in window && deadline instanceof window.IdleDeadline && deadline.timeRemaining() < minimum) {
```

Comment:
We don't need the `function` keyword anymore :)
Also this if statement is way to long. Can you refactor somewhat for readability? At least a line break.